com.unity.render-pipelines.high-definition/package.json

```diff
@@ -6,6 +6,9 @@
     "unityRelease": "14f1",
     "displayName": "High Definition RP",
     "dependencies": {
+        "com.unity.modules.video": "1.0.0",
+        "com.unity.modules.animation": "1.0.0",
```

Comment:
Is it expected that we do this change? sanity check

Reply:
I mean for 7.x. Want to be sure that those package above exist on 19.3

Reply:
The upm tests passed https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252F_projectcontext.yml%2523test_all_project_windows_2019.4/4793159/job so I'm guessing we're ok?